cilium/cmd/endpoint_list.go

```diff
@@ -44,7 +44,7 @@ func init() {
 
 func listEndpoint(w *tabwriter.Writer, ep *models.Endpoint, id string, label string) {
 	var isPolicyEnabled string
-	if ep.PolicyEnabled {
+	if *ep.PolicyEnabled {
```

Comment:
This is not safe. If it's a pointer then swagger will not guarantee that it is always set.

You can do:
```
import (
             "github.com/go-openapi/swag"
)

if swag.BoolValue(ep.PolicyEnabled) {
```